jodconverter-cli/src/integTest/java/org/jodconverter/cli/ConvertITest.java

```diff
@@ -187,6 +188,43 @@ public void convert_WithSingleFilter_ShouldSucceed() throws Exception {
   }
 
   @Test
+  public void main_Convert_WithConnectionOption_WithOutputFormat_ShouldSucceed() throws Exception {
+    final File inputFile = new File(SOURCE_FILE);
+    final File outputFile =
+        new File(
+            inputFile.getParentFile(), FilenameUtils.getBaseName(inputFile.getName()) + ".pdf");
+
+    assertThat(outputFile).doesNotExist();
```

Comment:
If for some reason the test fails after creating the file but before deleting it without throwing an exception the file is not deleted so this assertion will fail. See line 223.